src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Replace the literal '20' with a constant to represent the short-term rental discount. This will make your code more maintainable and align with checklist item 2.